src/Donation/Donation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Give\Donation;
+
+use function Give\Helpers\Form\Template\Utils\Frontend\getPaymentId;
+
+/**
+ * Class Donation
+ *
+ * @package Give\Donation
+ */
+class Donation {
+	private $id;
+
+	/**
+	 * Donation constructor.
+	 *
+	 * @param int|null $id
+	 */
+	public function __construct( $id = null ) {
+		$this->id = $id ?: getPaymentId();
+	}
+
+
+	/**
+	 * Return true if donation is subscription.
+	 *
+	 * @since 2.7.0
+	 * @return bool
+	 */
+	public function isRecurring() {
+		return '1' === Give()->payment_meta->get_meta( $this->id, '_give_is_donation_recurring', true );
```

Comment:
Should be broken into multiple statements for clarity. Readable code is better than fewer lines of code.

Reply:
This can actually be simplified as:
```suggestion
		return (bool) Give()->payment_meta->get_meta( $this->id, '_give_is_donation_recurring', true );
```

That to me reads pretty clearly.

Reply:
This is an unused file, so I delete this code.